android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebViewBinding.java

```diff
@@ -120,7 +120,7 @@ public void destroy()
 		destroyed = true;
 	}
 
-	private static StringBuilder readResourceFile(String fileName)
+	public static StringBuilder readResourceFile(String fileName)
```

Comment:
No longer needed if the `injectEventBindings()` logic is moved into this file.

```suggestion
	private static StringBuilder readResourceFile(String fileName)
```